src/stdlib_codata.f90

```diff
@@ -0,0 +1,1779 @@
+module stdlib_codata
+    !! Codata Constants - Autogenerated
```

Comment:
is the code to autogenerate this module available? Please provide a short explanation/reference.

Reply:
Yes the code is available. If needed, it can be incorporated in stdlib.

Reply:
OK. A short explanation and a link to the Python script should be sufficient. It is mainly to inform future developers.